### ISSUE TITLE
The new release is broken on sbt 0.13.11

### ISSUE THREAD

**Issue by omar-shibli-zocdoc** [localytics/sbt-sqs#6]
This is the error that I got (full stack trace below):
java.lang.NoSuchMethodError: com.localytics.sbt.PidUtils$.extractElasticMQPid(Ljava/lang/String;)Lscala/Option;

Workaround:
Downgrading to release 0.3.0 solved the issue.

Full stacktrace:
```
[info] Loading global plugins from /Users/omar.shibli/.sbt/0.13/plugins
Waiting for lock on /Users/omar.shibli/.ivy2/.sbt.ivy.lock to be available...
[info] Loading project definition from /Users/omar.shibli/workspace/my-project/document-service-flows/project
[info] Updating {file:/Users/omar.shibli/workspace/my-project/document-service-flows/project/}document-service-flows-build...
Waiting for lock on /Users/omar.shibli/.ivy2/.sbt.ivy.lock to be available...
[info] Resolving org.fusesource.jansi#jansi;1.4 ...
[info] Done updating.
[info] Loading project definition from /Users/omar.shibli/workspace/my-project/document-service-definitions/project
[info] Set current project to document-service-flows (in build file:/Users/omar.shibli/workspace/my-project/document-service-flows/)
[info] Compiling 14 Scala sources to /Users/omar.shibli/workspace/my-project/document-service-flows/target/scala-2.11/classes...
[info] Compiling 8 Scala sources to /Users/omar.shibli/workspace/my-project/document-service-flows/target/scala-2.11/test-classes...
[warn] ElasticMQ is already running at 0.0.0.0 on 31514
java.lang.NoSuchMethodError: com.localytics.sbt.PidUtils$.extractElasticMQPid(Ljava/lang/String;)Lscala/Option;
	at com.localytics.sbt.sqs.StartElasticMQ$.apply(StartElasticMQ.scala:33)
	at com.localytics.sbt.sqs.ElasticMQPlugin$$anonfun$projectSettings$10.apply(ElasticMQPlugin.scala:29)
	at com.localytics.sbt.sqs.ElasticMQPlugin$$anonfun$projectSettings$10.apply(ElasticMQPlugin.scala:29)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
[error] (document-service-flows/*:startElasticMq) java.lang.NoSuchMethodError: com.localytics.sbt.PidUtils$.extractElasticMQPid(Ljava/lang/String;)Lscala/Option;
[error] Total time: 32 s, completed Dec 21, 2016 10:42:08 PM
```

**Comment by bdarfler** [comment]
Hi @omar-shibli-zocdoc thanks for opening this issue.

Two questions come to mind:

1. What version of Scala are you using for the project?
2. Did you fully quit and reload sbt after updating the plugin?

Unfortunately, I am not able to reproduce this locally. I used this [bare bones sbt project](https://github.com/localytics/sbt-sqs/files/668923/test.zip) for testing, maybe you can give it a shot on your computer and see what results you get.

```
09:15 $ sbt
[info] Loading global plugins from /Users/Bdarfler/.sbt/0.13/plugins
[info] Loading project definition from /Users/Bdarfler/Desktop/test/project
[info] Updating {file:/Users/Bdarfler/Desktop/test/project/}test-build...
[info] Resolving org.fusesource.jansi#jansi;1.4 ...
[info] Done updating.
[info] Set current project to test (in build file:/Users/Bdarfler/Desktop/test/)
> startElasticMq
[info] Creating ElasticMQ directory elastic-mq
[info] Downloading ElasticMQ from [https://s3-eu-west-1.amazonaws.com/softwaremill-public/elasticmq-server-0.9.3.jar] to [/Users/Bdarfler/Desktop/test/elastic-mq/elasticmq-server-0.9.3.jar]
[info] Starting ElasticMQ
[info] Waiting for ElasticMQ to respond at '0.0.0.0' and port '9324'
09:16:22.569 [main] INFO  org.elasticmq.server.Main$ - Starting ElasticMQ server (0.9.3) ...
[info] Waiting for ElasticMQ to respond at '0.0.0.0' and port '9324'
09:16:22.895 [elasticmq-akka.actor.default-dispatcher-3] INFO  akka.event.slf4j.Slf4jLogger - Slf4jLogger started
[info] Waiting for ElasticMQ to respond at '0.0.0.0' and port '9324'
[info] Waiting for ElasticMQ to respond at '0.0.0.0' and port '9324'
09:16:23.816 [main] INFO  o.e.rest.sqs.TheSQSRestServerBuilder - Started SQS rest server, bind address 0.0.0.0:9324, visible server address http://localhost:9324
09:16:23.837 [main] INFO  org.elasticmq.server.Main$ - === ElasticMQ server (0.9.3) started in 1602 ms ===
[success] Total time: 11 s, completed Dec 22, 2016 9:16:24 AM
> stopElasticMq
[info] Stopping ElasticMQ
09:16:26.076 [Thread-0] INFO  org.elasticmq.server.Main$ - ElasticMQ server stopping ...
[success] Total time: 0 s, completed Dec 22, 2016 9:16:26 AM
09:16:26.107 [Thread-0] INFO  org.elasticmq.server.Main$ - === ElasticMQ server stopped ===
```

**Comment by bdarfler** [comment]
@omar-shibli-zocdoc I'm going to close this out if I don't hear back this week.

**Comment by omar-shibli-zocdoc** [comment]
Hey @bdarfler,

Yes, I did quit and clean up just in case.
I'm using Scala 2.11.8.
I just tried again a few moments ago, and the issue still persists.

Thanks,
Omar

**Comment by bdarfler** [comment]
@omar-shibli-zocdoc thanks for getting back in touch. Could you try the [bare bones sbt project](https://github.com/localytics/sbt-sqs/files/668923/test.zip) I mentioned above and see if you can reproduce it? As I said, I'm not able to reproduce the issue which is making this hard to debug.

**Comment by omar-shibli-zocdoc** [comment]
Thank you so much, i was able to run the plugin successfully with the bare bones sbt project, so it's probably an issue in my env. appreciate your help and thanks for your time again.

**Comment by bdarfler** [comment]
@omar-shibli-zocdoc best of luck! You might try deleting your `target`, `project/target`, and `project/project/target` directories.

**Comment by bdarfler** [comment]
@omar-shibli-zocdoc can you let me know if you are using any of the other localytics aws fakes? like localytics/sbt-s3 or localytics/sbt-dyanmodb? I have a hypothesis as to what is going on here.

**Comment by bdarfler** [comment]
@omar-shibli-zocdoc I believe I have fixed this with #7. Be sure to update any other `localytics/sbt-*` plugins you are using. Sorry for not understanding the root cause earlier.

**Comment by omar-shibli-zocdoc** [comment]
hey @bdarfler,

Yes, I'm using it with sbt-s3 plugin as well.
I just tested 0.4.1 and it works for me, I think your fix did the trick.

Thanks for the help,
Omar
